.github/workflows/tests.yml

```diff
@@ -42,6 +42,27 @@ jobs:
       - name: WCT local (Chrome/Firefox)
         run: npm test
 
+  wtr-local:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
```

Comment:
can we use `actions/setup-node@v2` and set `node-version: 14`?

Reply:
Will test whether it just works, didn't want to get sucked into a big task of debugging random infra stuff.